libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -547,10 +547,10 @@ void AP_MotorsHeli_Dual::servo_test()
         _pitch_test -= (1.0f / (_loop_rate/2));
         _oscillate_angle += 8 * M_PI / _loop_rate;
     } else if (_servo_test_cycle_time >= 5.0f && _servo_test_cycle_time < 6.0f){                              // Raise swash to top
-        _collective_test = 1.0f;
+        _collective_test += (1.0f / _loop_rate);
```

Comment:
@WickedShell Thanks for the comment.  Just as background, this is a start up test feature to enable the user to ensure there are no binding in the controls.  so it doesn't affect flight.  That being said, probably not a good idea to request more than the system is programmed to give and it would give the user a bad sense of the control limits.  Instead of doing this for each of these statements, I would put it at the end of the routine when it assigns the values to the global variables.

Reply:
That's a much better spot to put the constrain then my suggestion!